Admin/Admin.php

```diff
@@ -1240,7 +1241,7 @@ public function hasRoute($name)
     /**
      * {@inheritdoc}
      */
-    public function generateObjectUrl($name, $object, array $parameters = array(), $absolute = false)
+    public function generateObjectUrl($name, $object, array $parameters = array(), $absolute = UrlGeneratorInterface::ABSOLUTE_PATH)
```

Comment:
Well, as I can see, you just pass the value to the router. Should be BC AFAIK.

But, does `ABSOLUTE_PATH` constant exist?